src/lib/flows/create-donation/input-details.svelte

```diff
@@ -115,9 +129,46 @@
     bind:amount
     bind:selectedTokenAllowance
   />
+
+  {#if amount && selectedToken}
```

Comment:
I know this isn't the case yet and a question for later but: if and how does this change for ecosystems? Is there anything special or is it basically like a drip list?

Reply:
this should just work for ecosystems, since yeah, it's just an NFT Driver ID.